Entities/Materials/Ammunition/MaterialWaterArrow.cfg

```diff
@@ -19,7 +19,7 @@ $sprite_animation_start                = *start*
 $sprite_animation_default_name         = default
 u16 sprite_animation_default_time      = 0
 u8 sprite_animation_default_loop       = 0
-@u16 sprite_animation_default_frames   = 20; 28;
+@u16 sprite_animation_default_frames   = 20;
```

Comment:
Was this done because a full stack of 1 would show as the 2 water arrows sprite?  
If not this would be fine to keep so that it is easier for mods to change the max amount imo

Reply:
yeah, full stack of 1 shows 2 water arrows sprite without this